test/e2e-go/restAPI/restClient_test.go

```diff
@@ -1988,3 +1963,373 @@ int 1`
 	}
 	a.Equal(expectedResult, resp)
 }
+
+// The program is copied from pyteal source for c2c test over betanet:
+// source: https://github.com/ahangsu/c2c-testscript/blob/master/c2c_test/max_depth/app.py
+const maxDepthTealApproval = `#pragma version 8
```

Comment:
I just want to confirm, this program actually is the same as the linked pyteal one?

Reply:
This is exactly the same as the pyteal one, while the `simulate_eval_test.go` one is augmented and modified.